src/SprykerEco/Yves/Payone/Controller/ExpressCheckoutController.php

```diff
@@ -18,6 +18,7 @@ class ExpressCheckoutController extends AbstractController
     /**
      * @return array
      */
+    //phpcs:ignore
```

Comment:
* It's better to use `//phpcs:ignoreFile` at the beginning of the file.

Reply:
++